lib/WebSocket.js

```diff
@@ -98,10 +98,15 @@ class WebSocket extends EventEmitter {
   }
 
   set binaryType (type) {
-    if (type === 'arraybuffer' || type === 'nodebuffer') {
-      this._binaryType = type;
-    } else {
-      throw new SyntaxError('unsupported binaryType: must be either "nodebuffer" or "arraybuffer"');
+    if (constants.BINARY_TYPES.indexOf(type) < 0) {
+      throw new SyntaxError('unsupported binaryType: must be either "' +
```

Comment:
In chrome it ignores unsupported values but with a console.warn (see below)
Do you think we should do console.warn too?

![image](https://cloud.githubusercontent.com/assets/1091422/23342806/666e9c2c-fc69-11e6-85d0-0d9196500b37.png)